nix/modules/lanzaboote.nix

```
@@ -22,6 +22,8 @@ in
 
     enrollKeys = mkEnableOption "Automatic enrollment of the keys using sbctl";
 
+    generateKeysIfNotExist = mkEnableOption "Autogenerates the PKI bundle if it doesn't exist";
```

```suggestion
    generateKeysIfNotExist = mkEnableOption "autogeneration of the PKI bundle if it doesn't exist";
```

`mkEnableOption` prefixes the string you provide it with "Enable ".

---

Alright, looked it up. It gets templates with `"Whether to enable ${name}."`. Thanks for letting me know.